UI added for item information

ha2san opened the pull request:
new activity created and integrated. The activity gives information about the clicked item.
For future sprint, a link must be created between a ListViewModel and a Item

omarelmalki commented on app/src/main/java/com/github/brugapp/brug/ItemInformationActivity.kt:
```diff
@@ -0,0 +1,39 @@
+        icon.setImageResource(img)
+    }
+
+    @SuppressLint("SetTextI18n")
```
Why is this @SuppressLint annotation necessary ? I suggest adding comments to explain why is it necessary in the code.

omarelmalki commented on app/src/androidTest/java/com/github/brugapp/brug/ItemsMenuActivityTest.kt:
```diff
@@ -56,8 +59,9 @@ class ItemsMenuActivityTest {
     /* THESE NEED A CONFLICTING ESPRESSO DEPENDENCY */
     @Test
     fun swipeLeftOnItemTriggersSnackBar() {
+        Intents.init()
```
Do you plan on adding new tests that use Intents in the future ?
If yes, it would be great to put the Intents.init() and Intents.release() functions into a @Before  and @After function respectively, to avoid code duplication and conflicts. You can keep it in mind for next time or change it now as it is not a problem yet.

```suggestion
// (outside the class)
@Before
fun setUp() {
    Intents.init()
}
```

omarelmalki commented on app/src/androidTest/java/com/github/brugapp/brug/ItemsMenuActivityTest.kt:
```diff
@@ -66,7 +70,14 @@ class ItemsMenuActivityTest {
+                IntentMatchers.hasComponent(ItemInformationActivity::class.java.name)
+            )
+        )
+        Intents.release()
```
```suggestion
// (outside the class)
@After
fun finishUp() {
    Intents.release()
}
```

omarelmalki reviewed (commented):
Great work ! Feel free to check out my comments to improve documentation and tests.

ha2san commented:
thanks both of you, 
@DeepGreen1 I added some comments.
@omarelmalki I removed the code that was not necessary, and I keep in mind the remark you made to me for the next few times

hamzahass commented on app/src/androidTest/java/com/github/brugapp/brug/ItemInformationActivityTest.kt:
```diff
@@ -0,0 +1,38 @@
+            onView(ViewMatchers.withId(R.id.item_date)).check(matches(withText(str)))
+        }
+    }
+}
```
For the ItemInformationActivityTest, I would suggest to separate the checks into different Tests for future use, in case one of the fields fails the test it would be easier to point it out.

ha2san replied:
Thank you for your comment, I changed that as you requested in my last commit [fd6c7e1]

hamzahass commented on app/src/main/res/values/colors.xml:
```diff
@@ -7,4 +7,19 @@
+    <color name="followersBg">#65DCE5</color>
+    <color name="followingBg">#49BEFF</color>
+    <color name="grey">#E5E5E5</color>
+
```
Cirrus CI seems to say that some colors added on the colors.xml file are not actually used, in this case it would be good to delete these lines.

ha2san replied:
We may need these colors in the future, so it might be a good idea to keep them

hamzahass reviewed (commented):
Good work ! I have listed 2 comments below, please go over them and follow up on the concerns (ps : for the second comment on colors.xml file there is the same remark for the strings.xml file so please also check this file)
